src/screens/Lending/EnterWithdrawAmount.js

```diff
@@ -0,0 +1,232 @@
+
+
+type Props = {
+  depositedAssets: DepositedAsset[],
+  baseFiatCurrency: ?string,
+  rates: Rates,
+  navigation: NavigationScreenProp<*>,
+  isCalculatingWithdrawTransactionEstimate: boolean,
+  withdrawTransactionEstimate: ?Object,
+  calculateLendingWithdrawTransactionEstimate: (amount: number, asset: DepositedAsset) => void,
+  useGasToken: boolean,
+  balances: Balances,
+};
+
+const FeeInfo = styled.View`
+  align-items: center;
+  margin-bottom: ${spacing.large}px;
+`;
+
+const FooterInner = styled.View`
+  padding: ${spacing.large}px;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+`;
+
+const NotEnoughFee = styled(BaseText)`
+  margin-top: ${spacing.medium}px;
+`;
+
+const EnterWithdrawAmount = ({
+  navigation,
+  depositedAssets,
+  baseFiatCurrency,
+  rates,
+  withdrawTransactionEstimate,
+  isCalculatingWithdrawTransactionEstimate,
+  calculateLendingWithdrawTransactionEstimate,
+  useGasToken,
+  balances,
+}: Props) => {
+  const preselectedAssetSymbol: string = navigation.getParam('symbol');
+  const [selectedAssetSymbol, setSelectedAssetSymbol] = useState(preselectedAssetSymbol);
+  const [depositAmount, setDepositAmount] = useState(null);
+
+  const depositedAsset = depositedAssets.find(({ symbol }) => symbol === selectedAssetSymbol);
+
+  useEffect(() => {
+    if (!depositAmount || !depositedAsset) return;
+    calculateLendingWithdrawTransactionEstimate(depositAmount, depositedAsset);
+  }, [depositAmount, depositedAsset]);
+
+  const txFeeInfo = buildTxFeeInfo(withdrawTransactionEstimate, useGasToken);
+  const gasTokenSymbol = get(txFeeInfo?.gasToken, 'symbol', ETH);
+  const showTxFee = !!depositAmount && (!!txFeeInfo?.fee || isCalculatingWithdrawTransactionEstimate);
+  const isEnoughForFee = !!txFeeInfo?.fee && isEnoughBalanceForTransactionFee(balances, {
+    txFeeInWei: txFeeInfo.fee,
+    amount: depositAmount,
+    decimals: depositedAsset?.decimals,
+    symbol: selectedAssetSymbol,
+    gasToken: txFeeInfo.gasToken,
+  });
+
+  const showNextButton = depositAmount !== null; // only if amount input touched
+  const isNextButtonDisabled = !!isCalculatingWithdrawTransactionEstimate
+    || !depositAmount
+    || !isEnoughForFee
+    || (!!txFeeInfo?.fee && !txFeeInfo.fee.gt(0));
+  const nextButtonTitle = isCalculatingWithdrawTransactionEstimate ? 'Getting fee..' : 'Next';
+  const onNextButtonPress = () => navigation.navigate(
+    LENDING_WITHDRAW_TRANSACTION_CONFIRM,
+    { amount: depositAmount, asset: depositedAsset },
+  );
+
+  const assetSelectOptions = depositedAssets.reduce((assetsObject, asset) => ({
+    ...assetsObject,
+    [asset.symbol]: asset,
+  }), {});
+
+  const depositedAssetsBalances: Balances = depositedAssets.reduce(
+    (balancesObj, { currentBalance: balance, symbol }) => ({ ...balancesObj, [symbol]: { symbol, balance } }),
+    {},
+  );
+
+  const onValueChanged = (value: Object) => {
+    if (!value) {
+      if (depositAmount) setDepositAmount(0);
```

Comment:
I think the same here:

```suggestion
    if (!value && depositAmount) { setDepositAmount(0); }
```

Reply:
Replied in above.